pkg/cassandra/datacenter_test.go

```diff
@@ -357,6 +357,31 @@ func TestNewDatacenter_Fail_NoStorageConfig(t *testing.T) {
 	assert.IsType(t, DCConfigIncomplete{}, err)
 }
 
+func TestDatacentersReplication(t *testing.T) {
+	t.Run("EachDcContainsKeyspaces", testEachDcContainsKeyspaces)
```

Comment:
What's the point of this func? Why not simply capitalize the testEachDcContainsKeyspaces?

Reply:
I had multiple tests at one point. I'll ditch the subtest.